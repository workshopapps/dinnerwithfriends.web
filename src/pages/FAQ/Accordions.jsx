import React, { Fragment, useState } from "react";
import {
  Accordion,
  AccordionHeader,
  AccordionBody,
} from "@material-tailwind/react";

const Accordions = () => {
  const [open, setOpen] = useState(0);

  const handleOpen = (value) => {
    setOpen(open === value ? 0 : value);
  };

  function Icon({ id, open }) {
    return (
      <svg
        xmlns="http://www.w3.org/2000/svg"
        className={`${
          id === open ? "rotate-180" : ""
        } h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full `}
        fill="none"
        viewBox="0 0 24 24"
        stroke="currentColor"
        strokeWidth={2}
      >
        <path strokeLinecap="round" strokeLinejoin="round" d="M19 9l-7 7-7-7" />
      </svg>
    );
  }

  return (
    <Fragment>
      <Accordion open={open === 1} icon={<Icon id={1} open={open} />}>
        <AccordionHeader
          onClick={() => handleOpen(1)}
          className="!font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        >
          What is catch up?
        </AccordionHeader>
        <AccordionBody className="!text-[#616161] !text-[10px] lg:!text-sm">
          It is an invitation, gathering, hangout management tool. it enables
          you to easily send, manage, vote, and receive invitations seamlessly.
        </AccordionBody>
      </Accordion>
      <Accordion open={open === 2} icon={<Icon id={2} open={open} />}>
        <AccordionHeader
          onClick={() => handleOpen(2)}
          className="!font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        >
          How do i create an invitation?
        </AccordionHeader>
        <AccordionBody className="!text-[#616161] !text-[10px] lg:!text-sm">
          Please click on the New (+) at the bottom of the screen to start a new
          invitation or request a vote on time or location, fill all required
          fileds, and make sure to have at least one person invited to complete
          the process.
        </AccordionBody>
      </Accordion>
      <Accordion open={open === 3} icon={<Icon id={3} open={open} />}>
        <AccordionHeader
          onClick={() => handleOpen(3)}
          className="!font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff] !text-left"
        >
          Can I change my mind about an invitation I received?
        </AccordionHeader>
        <AccordionBody className="!text-[#616161] !text-[10px] lg:!text-sm">
          Yes, you can reselect another reply to your received invitations if
          the event did not start yet. Once the event starts, you cannot change
          your reply
        </AccordionBody>
      </Accordion>
      <Accordion open={open === 4} icon={<Icon id={4} open={open} />}>
        <AccordionHeader
          onClick={() => handleOpen(4)}
          className="!font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        >
          Who should use catchup?
        </AccordionHeader>
        <AccordionBody className="!text-[#616161] !text-[10px] lg:!text-sm">
          Anyone struggling with their social life, trying to figure out what
          works for them and their guest/friends. Now they can instantly find
          out when their friends are all free and make plans on the spot.
        </AccordionBody>
      </Accordion>
      <Accordion open={open === 5} icon={<Icon id={5} open={open} />}>
        <AccordionHeader
          onClick={() => handleOpen(5)}
          className="!font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        >
          How do I get help?
        </AccordionHeader>
        <AccordionBody className="!text-[#616161] !text-[10px] lg:!text-sm">
          We have a live chat, offering support as fast as possible. Otherwise,
          you can always email us at support@catchup.com, and will get back to
          you within 24 hours.
        </AccordionBody>
      </Accordion>
      <Accordion open={open === 6} icon={<Icon id={6} open={open} />}>
        <AccordionHeader
          onClick={() => handleOpen(6)}
          className="!font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        >
          Can I add an event to my calendar?
        </AccordionHeader>
        <AccordionBody className="!text-[#616161] !text-[10px] lg:!text-sm">
          Catch up would ask to get permission to your calender, If you allowed
          access, then events will be added to your calendar using the time zone
          of the location of the even
        </AccordionBody>
      </Accordion>
      <Accordion open={open === 7} icon={<Icon id={7} open={open} />}>
        <AccordionHeader
          onClick={() => handleOpen(7)}
          className="!font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff] !text-left"
        >
          Do I need to download the app to create or receive invite?
        </AccordionHeader>
        <AccordionBody className="!text-[#616161] !text-[10px] lg:!text-sm">
          No, you donâ€™t need to download to create an invite. With access to
          your mobile phone or desktop that has internet you can utilize all the
          fuctions and features of the app.
        </AccordionBody>
      </Accordion>
      <Accordion open={open === 8} icon={<Icon id={8} open={open} />}>
        <AccordionHeader
          onClick={() => handleOpen(8)}
          className="!font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        >
          Can I invite friends from my contact list?
        </AccordionHeader>
        <AccordionBody className="!text-[#616161] !text-[10px] lg:!text-sm">
          Catch up would ask to get permission from your contact list, if
          contact is allowed access, then you can add your contact to catch up.
        </AccordionBody>
      </Accordion>
    </Fragment>
  );
};

export default Accordions;
