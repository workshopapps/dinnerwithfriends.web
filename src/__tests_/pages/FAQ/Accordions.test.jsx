import { render } from '@testing-library/react';
import Accordions from '../../../pages/FAQ/Accordions';
import { BrowserRouter } from 'react-router-dom';

describe('Accordion component', () => {
  test('expect accordion component to render correctly', () => {
    // eslint-disable-next-line testing-library/render-result-naming-convention
    const faq = render(
        <BrowserRouter>
          <Accordions />
        </BrowserRouter>
    );
    expect(faq).toMatchSnapshot();
  });
});
