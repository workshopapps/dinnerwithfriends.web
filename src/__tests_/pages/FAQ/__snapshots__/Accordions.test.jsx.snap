// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accordion component expect faq page to render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="block relative w-full"
      >
        <button
          class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
          type="button"
        >
          What is catch up?
          <span
            class="w-5 h-5 ml-4"
          >
            <svg
              class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
              fill="none"
              stroke="currentColor"
              stroke-width="2"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M19 9l-7 7-7-7"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </span>
        </button>
        <div
          class="overflow-hidden"
          style="height: 0px;"
        >
          <div
            class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
            style="opacity: 0;"
          >
            It is an invitation, gathering, hangout management tool. it enables you to easily send, manage, vote, and receive invitations seamlessly.
          </div>
        </div>
      </div>
      <div
        class="block relative w-full"
      >
        <button
          class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
          type="button"
        >
          How do i create an invitation?
          <span
            class="w-5 h-5 ml-4"
          >
            <svg
              class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
              fill="none"
              stroke="currentColor"
              stroke-width="2"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M19 9l-7 7-7-7"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </span>
        </button>
        <div
          class="overflow-hidden"
          style="height: 0px;"
        >
          <div
            class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
            style="opacity: 0;"
          >
            Please click on the New (+) at the bottom of the screen to start a new invitation or request a vote on time or location, fill all required fileds, and make sure to have at least one person invited to complete the process.
          </div>
        </div>
      </div>
      <div
        class="block relative w-full"
      >
        <button
          class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff] !text-left"
          type="button"
        >
          Can I change my mind about an invitation I received?
          <span
            class="w-5 h-5 ml-4"
          >
            <svg
              class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
              fill="none"
              stroke="currentColor"
              stroke-width="2"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M19 9l-7 7-7-7"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </span>
        </button>
        <div
          class="overflow-hidden"
          style="height: 0px;"
        >
          <div
            class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
            style="opacity: 0;"
          >
            Yes, you can reselect another reply to your received invitations if the event did not start yet. Once the event starts, you cannot change your reply
          </div>
        </div>
      </div>
      <div
        class="block relative w-full"
      >
        <button
          class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
          type="button"
        >
          Who should use catchup?
          <span
            class="w-5 h-5 ml-4"
          >
            <svg
              class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
              fill="none"
              stroke="currentColor"
              stroke-width="2"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M19 9l-7 7-7-7"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </span>
        </button>
        <div
          class="overflow-hidden"
          style="height: 0px;"
        >
          <div
            class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
            style="opacity: 0;"
          >
            Anyone  struggling with their social life, trying to figure  out what works for them and their guest/friends. Now they can instantly find out when their friends are all free and make plans on the spot.
          </div>
        </div>
      </div>
      <div
        class="block relative w-full"
      >
        <button
          class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
          type="button"
        >
          How do I get help?
          <span
            class="w-5 h-5 ml-4"
          >
            <svg
              class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
              fill="none"
              stroke="currentColor"
              stroke-width="2"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M19 9l-7 7-7-7"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </span>
        </button>
        <div
          class="overflow-hidden"
          style="height: 0px;"
        >
          <div
            class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
            style="opacity: 0;"
          >
            We have a live chat, offering support as fast as possible. Otherwise, you can always email us at support@catchup.com, and will get back to you within 24 hours.
          </div>
        </div>
      </div>
      <div
        class="block relative w-full"
      >
        <button
          class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
          type="button"
        >
          Can I add an event to my calendar?
          <span
            class="w-5 h-5 ml-4"
          >
            <svg
              class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
              fill="none"
              stroke="currentColor"
              stroke-width="2"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M19 9l-7 7-7-7"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </span>
        </button>
        <div
          class="overflow-hidden"
          style="height: 0px;"
        >
          <div
            class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
            style="opacity: 0;"
          >
            Catch up would ask to get permission to your calender, If you allowed access, then events will be added to your calendar using the time zone of the location of the even
          </div>
        </div>
      </div>
      <div
        class="block relative w-full"
      >
        <button
          class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff] !text-left"
          type="button"
        >
          Do I need to download the app to create or receive invite?
          <span
            class="w-5 h-5 ml-4"
          >
            <svg
              class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
              fill="none"
              stroke="currentColor"
              stroke-width="2"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M19 9l-7 7-7-7"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </span>
        </button>
        <div
          class="overflow-hidden"
          style="height: 0px;"
        >
          <div
            class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
            style="opacity: 0;"
          >
            No, you don’t need to download  to create an invite. With access to your mobile phone or desktop that has internet you can utilize all the fuctions and features of the app.
          </div>
        </div>
      </div>
      <div
        class="block relative w-full"
      >
        <button
          class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
          type="button"
        >
          Can I invite friends from my contact list?
          <span
            class="w-5 h-5 ml-4"
          >
            <svg
              class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
              fill="none"
              stroke="currentColor"
              stroke-width="2"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M19 9l-7 7-7-7"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </span>
        </button>
        <div
          class="overflow-hidden"
          style="height: 0px;"
        >
          <div
            class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
            style="opacity: 0;"
          >
            Catch up would ask to get permission from your contact list, if contact is allowed access, then you can add your contact to catch up.
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="block relative w-full"
    >
      <button
        class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        type="button"
      >
        What is catch up?
        <span
          class="w-5 h-5 ml-4"
        >
          <svg
            class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
            fill="none"
            stroke="currentColor"
            stroke-width="2"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 9l-7 7-7-7"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </span>
      </button>
      <div
        class="overflow-hidden"
        style="height: 0px;"
      >
        <div
          class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
          style="opacity: 0;"
        >
          It is an invitation, gathering, hangout management tool. it enables you to easily send, manage, vote, and receive invitations seamlessly.
        </div>
      </div>
    </div>
    <div
      class="block relative w-full"
    >
      <button
        class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        type="button"
      >
        How do i create an invitation?
        <span
          class="w-5 h-5 ml-4"
        >
          <svg
            class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
            fill="none"
            stroke="currentColor"
            stroke-width="2"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 9l-7 7-7-7"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </span>
      </button>
      <div
        class="overflow-hidden"
        style="height: 0px;"
      >
        <div
          class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
          style="opacity: 0;"
        >
          Please click on the New (+) at the bottom of the screen to start a new invitation or request a vote on time or location, fill all required fileds, and make sure to have at least one person invited to complete the process.
        </div>
      </div>
    </div>
    <div
      class="block relative w-full"
    >
      <button
        class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff] !text-left"
        type="button"
      >
        Can I change my mind about an invitation I received?
        <span
          class="w-5 h-5 ml-4"
        >
          <svg
            class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
            fill="none"
            stroke="currentColor"
            stroke-width="2"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 9l-7 7-7-7"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </span>
      </button>
      <div
        class="overflow-hidden"
        style="height: 0px;"
      >
        <div
          class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
          style="opacity: 0;"
        >
          Yes, you can reselect another reply to your received invitations if the event did not start yet. Once the event starts, you cannot change your reply
        </div>
      </div>
    </div>
    <div
      class="block relative w-full"
    >
      <button
        class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        type="button"
      >
        Who should use catchup?
        <span
          class="w-5 h-5 ml-4"
        >
          <svg
            class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
            fill="none"
            stroke="currentColor"
            stroke-width="2"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 9l-7 7-7-7"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </span>
      </button>
      <div
        class="overflow-hidden"
        style="height: 0px;"
      >
        <div
          class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
          style="opacity: 0;"
        >
          Anyone  struggling with their social life, trying to figure  out what works for them and their guest/friends. Now they can instantly find out when their friends are all free and make plans on the spot.
        </div>
      </div>
    </div>
    <div
      class="block relative w-full"
    >
      <button
        class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        type="button"
      >
        How do I get help?
        <span
          class="w-5 h-5 ml-4"
        >
          <svg
            class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
            fill="none"
            stroke="currentColor"
            stroke-width="2"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 9l-7 7-7-7"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </span>
      </button>
      <div
        class="overflow-hidden"
        style="height: 0px;"
      >
        <div
          class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
          style="opacity: 0;"
        >
          We have a live chat, offering support as fast as possible. Otherwise, you can always email us at support@catchup.com, and will get back to you within 24 hours.
        </div>
      </div>
    </div>
    <div
      class="block relative w-full"
    >
      <button
        class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        type="button"
      >
        Can I add an event to my calendar?
        <span
          class="w-5 h-5 ml-4"
        >
          <svg
            class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
            fill="none"
            stroke="currentColor"
            stroke-width="2"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 9l-7 7-7-7"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </span>
      </button>
      <div
        class="overflow-hidden"
        style="height: 0px;"
      >
        <div
          class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
          style="opacity: 0;"
        >
          Catch up would ask to get permission to your calender, If you allowed access, then events will be added to your calendar using the time zone of the location of the even
        </div>
      </div>
    </div>
    <div
      class="block relative w-full"
    >
      <button
        class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff] !text-left"
        type="button"
      >
        Do I need to download the app to create or receive invite?
        <span
          class="w-5 h-5 ml-4"
        >
          <svg
            class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
            fill="none"
            stroke="currentColor"
            stroke-width="2"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 9l-7 7-7-7"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </span>
      </button>
      <div
        class="overflow-hidden"
        style="height: 0px;"
      >
        <div
          class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
          style="opacity: 0;"
        >
          No, you don’t need to download  to create an invite. With access to your mobile phone or desktop that has internet you can utilize all the fuctions and features of the app.
        </div>
      </div>
    </div>
    <div
      class="block relative w-full"
    >
      <button
        class="flex justify-between items-center w-full py-4 border-b border-b-blue-gray-100 text-blue-gray-700 antialiased font-sans text-xl font-semibold leading-snug select-none hover:text-blue-gray-900 transition-colors !font-bold !text-xs lg:!text-xl !text-[#151517] !border-0 !border-[#fff]"
        type="button"
      >
        Can I invite friends from my contact list?
        <span
          class="w-5 h-5 ml-4"
        >
          <svg
            class=" h-5 w-5 transition-transform text-[#0056D6] border border-solid border-[#0056D6] p-1 rounded-full "
            fill="none"
            stroke="currentColor"
            stroke-width="2"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 9l-7 7-7-7"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
        </span>
      </button>
      <div
        class="overflow-hidden"
        style="height: 0px;"
      >
        <div
          class="block w-full py-4 text-gray-700 antialiased font-sans text-sm font-light leading-normal !text-[#616161] !text-[10px] lg:!text-sm"
          style="opacity: 0;"
        >
          Catch up would ask to get permission from your contact list, if contact is allowed access, then you can add your contact to catch up.
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
